presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -103,7 +104,7 @@
     // fields and expressions in the group by clause
     private final Set<FieldId> groupingFields;
     private final List<Expression> expressions;
-    private final Map<NodeRef<Expression>, FieldId> columnReferences;
+    private final ListMultimap<NodeRef<Expression>, FieldId> columnReferences;
```

Comment:
We normally use the most generic type. So use `MultiMap` here instead.

Reply:
Ok, changed all the ListMultimap to Multimap.